backend/src/main/java/com/codesquad/sidedish/dish/dto/DishResponse.java

```diff
@@ -0,0 +1,43 @@
+package com.codesquad.sidedish.dish.dto;
+
+import com.codesquad.sidedish.dish.domain.Dish;
+import com.codesquad.sidedish.dish.domain.DishImage;
+import com.codesquad.sidedish.event_badge.dto.EventBadgeResponse;
+import java.util.List;
+import java.util.stream.Collectors;
+import lombok.Getter;
+import lombok.RequiredArgsConstructor;
+
+@Getter
+@RequiredArgsConstructor
+public class DishResponse {
+
+    private final Integer dishId;
+    private final String title;
+    private final String description;
+    private final Integer price;
+    private final Integer stock;
+
+    private final List<EventBadgeResponse> eventBadges;
+    // dish image
+    private final String imagePath;
+
+    public static DishResponse from(Dish dish) {
+        List<EventBadgeResponse> eventBadges = dish.getEventBadges() != null ?
+            dish.getEventBadges().stream()
+                .map(EventBadgeResponse::from)
+                .collect(Collectors.toList()) : null;
```

Comment:
```suggestion
        List<EventBadgeResponse> eventBadges = (dish.getEventBadges() != null)
            ? dish.getEventBadges().stream()
                .map(EventBadgeResponse::from)
                .collect(Collectors.toList()) 
            : null;
```

- 개인적으로 삼항연산자는 가독성의 문제가 있기에 1줄로 표현하기 힘든 것에서는 사용하지 않습니다.
- dish.getEventBadges()는 List로 보이는데 null이 아니라 emptyList가 될 수 있도록 하는게 nullsafe하지 않을까요?

Reply:
지금까지 null 과 empty 의 차이에 대해 깊게 생각해본 적 없었던 것 같습니다.
서버에서는 nullsafe 한 객체를 만들고, 클라이언트에서도 응답에 대해 null 체크 필요없이 프로그래밍이 가능하므로,
수정하도록 하겠습니다.

(아래의 링크를 참고하겠습니다.)

<https://stackoverflow.com/questions/40392820/how-to-handle-null-pointer-and-what-is-null-safe-way-to-code>
<https://stackoverflow.com/questions/9619852/what-is-the-convention-in-json-for-empty-vs-null>